src/content/tools/pub/dependencies.md

```diff
@@ -432,6 +432,30 @@ As a result, if you publish a package to pub.dev,
 keep in mind that your package's dependency overrides
 are ignored by all users of your package.
 
+If you are using a [pub workspace][workspaces], you can place `dependency_overrides` in all
+workspace packages, but a single package can only be overridden once in the
+workspace.
+
+## `pubspec_overrides.yaml`
+
+If you want to change certain aspects of your `pubspec.yaml` but do not
+want to have active changes to the actual file, you can place a file called
```

Comment:
Could you expand on what you meant by active in "active changes to the actual file"?

Do you mean changes that would affect consumers of the package?

Reply:
I mean something like "active version control changes" but I tried to simplify, just focus on avoiding changing the main file.